frontend/src/app/tasks/month/page.tsx

```diff
@@ -1,3 +1,134 @@
-export default function MonthView() {
-  return <div>Month!</div>
+import {
+  startOfMonth,
+  startOfWeek,
+  lastDayOfMonth,
+  lastDayOfWeek,
+  format,
+  eachDayOfInterval,
+  isSameDay,
+  isSameMonth
+} from 'date-fns'
+import { Box, Typography } from '@mui/joy'
+import { getTasksWithDate } from '../actions/getTasksWithDate'
+import { convertMinutesToTime } from '../utils/time'
+import { SimpleTaskBox } from '../components/TaskBox'
+import { Fragment } from 'react'
+
+const WEEK_DAYS = ['Sunday', 'Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday']
+
+export default async function MonthView() {
+  const today = new Date()
+  const firstDayOfFirstWeek = startOfWeek(startOfMonth(today))
+  const lastDayOfLastWeek = lastDayOfWeek(lastDayOfMonth(today))
+  const dateRange = eachDayOfInterval({ start: firstDayOfFirstWeek, end: lastDayOfLastWeek })
+
+  const groupedTasks = await getTasksWithDate(
+    format(firstDayOfFirstWeek, 'yyyy-MM-dd'),
+    format(lastDayOfLastWeek, 'yyyy-MM-dd'),
+    500
+  )
+
+  return (
+    <Box
```

Comment:
Right above the grid holding the month, what do you think about adding an `<h1>` centered above it with the name of the month? It can be good to make sure users know where/when they are. And once we add the date picker at the top, I imagine it would be more important to have the month as a wayfinding helper.  I don't have a strong preference on full month name  or three-letter abbreviation. 

It would probably be nice on the week page, too: 
- January 21, 2023 - January 27, 2023  or 
- Jan 21, 2023 - Jan 27, 2023  or
- Jan 21, 2023 - Jan 27, 2023 (W4)

Having headings is also a good way to help users who rely on assistive devices to know what is on a page. Right now, the page title of the app only ever reads 'PhpReport' which is not terribly informative, so having a heading level 1 with the page title would be useful.

![Screenshot from 2024-01-24 11-28-51](https://github.com/Igalia/phpreport/assets/3666105/5c4d788b-77a3-4de9-9bd2-7c4b74a05e96)

Reply:
Week:

![image](https://github.com/Igalia/phpreport/assets/20492786/49d99357-674e-46a0-8f80-8debaf166145)

Month:

![image](https://github.com/Igalia/phpreport/assets/20492786/58fdb709-6e0f-41cd-b95d-651a90383458)